netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -39,24 +39,14 @@ typedef struct _net_ebpf_ext_connect_context_address_info
     uint16_t source_port;
 } net_ebpf_ext_connect_context_address_info_t;
 
-typedef struct _net_ebpf_ext_connection_context_key
```

Comment:
I think we should keep the key separate. There is another issue #1675 for changing linked list to hash table. That will again require creating the key structure.

Reply:
I dont see how that is an issue. If the current structure can be used in a linked list, it can be used in a hashtable as well.